src/plugins/query_enhancements/server/connections/managers/prometheus_manager.ts

```diff
@@ -14,12 +14,14 @@ import { URI } from '../../../common/constants';
 import { PromQLConnectionClient } from '../clients/promql_connection_client';
 
 const BASE_RESOURCE_API = 'api/v1';
+const BASE_ALERT_MANAGER_API = 'alertmanager/api/v2';
```

Comment:
Looks like this route depends on the alert manager plugin. The intention here was to be a proxy to the Prometheus resource APIs. This doesn't look to align with that and seems better served in alert manager given these aren't concepts of Prometheus itself, but rather the alert manager plugin.

Reply:
This doesn’t create a dependency on Alertmanager but aligns with our design where Alertmanager data is part of the resources. This API serves as the central place for all Prometheus resource data. Keeping it separate in the alerting plugin would lead to duplication and unnecessary dependencies, requiring other modules to rely on the alerting plugin if they also need any data from Alertmanager outside of alerting plugin.

Reply:
Gotcha makes sense that this isn't coming from the actual alerts plugin. But why are we hitting a different base api than our existing ${BASE_RESOURCE_API}/alerts then?
[https://github.com/opensearch-project/OpenSearch-Dashboards/blob/feature/prometheu[…]_enhancements/server/connections/managers/prometheus_manager.ts](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/feature/prometheus/src/plugins/query_enhancements/server/connections/managers/prometheus_manager.ts#L78)
I guess why conceptually do we want an different base api only for /alerts/groups than all the other resources?

Reply:
Followup: This is actually based on prometheus's api structure that we directly proxy through to the prometheus server in the sql plugin. Makes sense then.